catkit2/services/hamamatsu_camera/hamamatsu_camera.py

```diff
@@ -270,8 +261,90 @@ def monitor_temperature(self):
             temperature = self.get_temperature()
             self.temperature.submit_data(np.array([temperature]))
 
+            if temperature > 30:
+                self.log.warning('Camera Temperature above 30 degrees, stopping acquisition and start fan')
+                self.fan_status = 'on'
+                self.should_shut_down = True

```

Comment:
- Changing the shut down flag to true doesn't just stop the camera, it stops the entire service. Is this intended? If yes, please change the warning message, if not, please adapt the code.
- Should this also change the cooler mode, or is it sufficient to just turn the fan on?